packages/plugins/vcard/index.js

```diff
@@ -0,0 +1,55 @@
+'use strict'
+
+const xml = require('@xmpp/xml')
+const iqCaller = require('../iq-caller')
+
+const NS_VCARD = 'vcard-temp'
+
+const parsevCard = (el) => {
+  const dict = {}
+  el.children.forEach((c) => {
+    if (c.children && typeof c.children[0] === 'string') {
+      dict[c.name] = c.text()
+    } else {
+      dict[c.name] = parsevCard(c)
+    }
+  })
+  return dict
+}
+
+const buildvCard = (dict, parent) => {
+  const builder = parent || xml`<vCard xmlns="${NS_VCARD}" version="2.0"/>`
+  for (const [key, val] of Object.entries(dict)) {
```

Comment:
travis is failing, :( `Object.entries` is Node.js >= 7 only 

We support Node.js >= 4

I'll configure eslint accordingly.